source/installguide/management-server/_database.rst

```diff
@@ -197,6 +197,17 @@ MySQL. See :ref:`install-database-on-separate-node`.
       cluster management server node IP. If not specified, the local IP
       address will be used.
 
+   -  (Optional) The cloudstack-setup-databases script is responsible for
+      creating the cloudstack databases (cloud, cloud_usage), creating a
+      user (cloud), granting permissions to the user and preparing the tables
```

Comment:
can you add the permissions details to be granted for 'cloud' user

Reply:
Sure, np